app/v1/connectors/CreateAndAmendEmploymentExpensesConnector.scala

```diff
@@ -29,15 +30,21 @@ import scala.concurrent.{ExecutionContext, Future}
 @Singleton
 class CreateAndAmendEmploymentExpensesConnector @Inject() (val http: HttpClient, val appConfig: AppConfig) extends BaseDownstreamConnector {
 
-  def amend(request: CreateAndAmendEmploymentExpensesRequest)(implicit
-      hc: HeaderCarrier,
-      ec: ExecutionContext,
-      correlationId: String): Future[DownstreamOutcome[Unit]] = {
+  def createAmendEmploymentExpenses(request: CreateAndAmendEmploymentExpensesRequest)(implicit
+                                                                                   hc: HeaderCarrier,
+                                                                                   ec: ExecutionContext,
+                                                                                   correlationId: String): Future[DownstreamOutcome[Unit]] = {
+
+    import request._
 
-    put(
-      body = request.body,
-      uri = IfsR6Uri[Unit](s"income-tax/expenses/employments/${request.nino}/${request.taxYear}")
-    )
+    val downstreamUri = if (taxYear.useTaxYearSpecificApi) {
+      TaxYearSpecificIfsUri[Unit](s"income-tax/${taxYear.asTysDownstream}/expenses/employments/${nino.value}")
```

Comment:
the nino part can just be `$nino` - as its toString returns the value.

Reply:
So is it a general thing that the toString method will just get called if you are collecting the value of an instance of the case class?